src/core/awspack/AbstractTextToSpeechFeature.js

```diff
@@ -725,53 +725,9 @@ class AbstractTextToSpeechFeature extends AbstractHostFeature {
             }
 
             markTypes[mark.type].push(mark);
-            endMarkTypes[mark.type] = mark;
             return mark;
           }
         );
-
-        // Find the time of the latest speechmark
-        const endTimes = [];

```

Comment:
These code are used to set the duration of the last speech mark since in previous mapping function `lastMark.duration = mark.time - lastMark.time;` durations are set for all speech mark except the last one